index.js

```diff
@@ -464,6 +475,7 @@
 			return Q(forceupdate)
 				.then(preCallHook.bind(this))
 				.then(applyChangesAndValidateCall.bind(this))
+				.then(customValidation.bind(this))
```

Comment:
probably not in scope for this feature, but just something I'd note as a to do is consolidate function call validation...right now we have different validation steps smeared across `applyChangesAndValidateCall`, `customValidation`, and `callIfValid`. I put a ticket in case we want to do this later `17660`.